tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -203,9 +215,10 @@ void ROOT::Experimental::Internal::RNTupleMerger::Merge(std::span<RPageSource *>
       auto clusterId = descriptor->FindClusterId(0, 0);
 
       while (clusterId != ROOT::Experimental::kInvalidDescriptorId) {
-         auto &cluster = descriptor->GetClusterDescriptor(clusterId);
+         auto *clusterP = clusterPool.GetCluster(clusterId, columnSet);
+         assert(clusterP);
+         const auto &cluster = descriptor->GetClusterDescriptor(clusterId);
```

Comment:
Perhaps for clarity:
```suggestion
         auto *clusterPool = clusterPool.GetCluster(clusterId, columnSet);
         assert(clusterPool);
         const auto &clusterDesc = descriptor->GetClusterDescriptor(clusterId);
```

Reply:
I guess you meant `cluster` rather than `clusterPool`, since `clusterPool` already exists

Reply:
Oh. Yes :)